byterun/interp.c

```diff
@@ -723,6 +723,8 @@ value caml_interprete(code_t prog, asize_t prog_size)
       accu = Field(accu, *pc); pc++; Next;
     Instruct(GETFLOATFIELD): {
       double d = Double_field(accu, *pc);
+      /* CR doligez: change to Double_flat_field ? Need to check how the
+         compiler uses this instruction. */
```

Comment:
I think these are only used for accesses to values known to have tag `Double_array_tag`.  (So yes.)